### DISCUSSION TITLE
Please add a link from the Bepacom WebUI to the Add-On Info view

### DISCUSSION THREAD

**Discussion by CV-Private** [Bepacom-Raalte/bepacom-HA-Addons#51] [Ideas]
There is a link from the Add-On Info view to the WebUI:

![image](https://github.com/Bepacom-Raalte/bepacom-HA-Addons/assets/108861703/847bd6d4-923d-46d1-a53f-bfb517305371)

However, there's unfortunately no direct link in the other direction. To open the Add-On Info view, one has to go all the way back to Settings => Add-Ons and so forth. Those are a lot of clicks if one simply wants to open the Add-On Configuration view from the WebUI.

Could you please add a direct link from the Bepacom WebUI to the Add-On Info view?

**Comment by GravySeal** [comment]
I added a button for this in the latest dev version.

**Comment by CV-Private** [comment]
I found the button, but the link brings up the spinning wheel that never ends. 
I noticed that the correct link to the Configuration view is "_http://192.168.xx.xx:8123/hassio/addon/97683af0_bacnetinterface_dev/info_", but the button links to "_http://192.168.xx.xx:8123/97683af0_bacnetinterface_dev/addon_". 
I think you should add "_../hassio/addon.._" to the button-link.
